app/webpacker/controllers/checkbox_selection_controller.js

```diff
@@ -0,0 +1,40 @@
+import { Controller } from "stimulus";
+
+export default class extends Controller {
+  static targets = ["checkbox", "errorMessage"]
+
+  get disableElements() {
+    return document.querySelectorAll("[data-checkbox-selection-button]")
+  }
+
+  connect() {
+    this.handleError(false)
+  }
+
+  disableButtons() {
+    const groupStates = this.checkboxTargets.reduce((acc, element) => {
+     const group = element.getAttribute("data-group")
```

Comment:
You can also use `.dataset`

```suggestion
     const { group } = element.dataset
```